core/trace/src/main/scala/org/typelevel/otel4s/trace/SpanContext.scala

```diff
@@ -80,4 +81,6 @@ object SpanContext {
       val isRemote: Boolean = false
     }
 
+  def fromContext(context: Vault): Option[SpanContext] =
+    Span.fromContext[cats.Id](context).map(_.context)
```

Comment:
This `Id` is nonsense, but we don't actually care what kind of span it is.  The alternative is to pollute the API with a dummy F[_].